praw/util/token_manager.py

```diff
@@ -59,7 +59,18 @@ def pre_refresh_callback(self, authorizer):
 
 
 class FileTokenManager(BaseTokenManager):
-    """Provides a trivial single-file based token manager."""
+    """Provides a trivial single-file based token manager.
+
+    It is expected that the file with the initial ``refresh_token`` is created prior to use.
```

Comment:
```suggestion
    It is expected that the file with the initial ``refresh_token`` is created prior to
    use.
```
To make docstrfmt happy.